src/__tests__/firestore.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @jest-environment node
+ */
+
+import { firebase } from '../tests/testUtils'
+
+import type { TokenOptions } from '@firebase/rules-unit-testing/dist/src/api'
+
+// eslint-disable-next-line prettier/prettier
+const PROJECT_ID = "raid-stats-c1d5a"
+
+function getAuthFirestore(auth?: TokenOptions) {
+  const db = firebase
+    .initializeTestApp({ projectId: PROJECT_ID, auth })
+    .firestore()
+
+  // The following line is for this error
+  // "[Warning: FIRESTORE_EMULATOR_HOST not set, using default value localhost:8080]"
+  db.useEmulator('localhost', 8080)
+  return db
+}
+
+test('Should Succeed: SET - with Auth', async () => {
+  const db = getAuthFirestore({ uid: 'alice' })
+
+  const usersRef = db.collection('users').doc('alice')
+  await firebase.assertSucceeds(usersRef.get())
```

Comment:
IMO this whole suite isn't all that useful...

But now that it is here: This test says it is testing SET - but you call get? Either way, this should FAIL - the rules are setup so that nobody can write to the database.

As an aside, we aren't even using the firebase auth (yet)